src/main/resources/assets/js/application/recruiter/tdprRecruiterModuleConfig.js

```diff
@@ -21,6 +21,13 @@ define(['angular'
                     controller: "tdprWeekTableController"
                 }
             }
+        }).state("tdpr.recruiter.add", {
```

Comment:
this state name suggests its purpose is to add recruiters. Maybe change that to addInterviewer?